File: WebApi/TinyUrl.Core/DataTransferObjects/UrlDto.cs
namespace TinyUrl.Core.DataTransferObjects
{
    public class UrlDto
    {
        public string Alias { get; set; }
        public string OriginalUrl { get; set; }
        public string ShortUrl { get; set; }
        public DateTime UrlCreated { get; set; }
    }
}

File: ReactApp/tiny-url/src/models/dto/url.dto.js
export default class UrlDto {
    originalUrl = "";
    shortUrl = "";
    alias = "";
    urlCreated = null;

    constructor(originalUrl, shortUrl, alias, urlCreated) {
        this.originalUrl = originalUrl;
        this.shortUrl = shortUrl;
        this.alias = alias;
        this.urlCreated = urlCreated;
    }

    static fromResponse(response) {
        return new UrlDto(
            response.originalUrl,
            response.shortUrl,
            response.alias,
            response.urlCreated);
    }
}